src/main/java/carpet/script/CarpetEventServer.java

```diff
@@ -983,11 +982,9 @@ public static String getEntityLoadEventName(EntityType<? extends Entity> et)
         }
 
         @Deprecated
-        public static final Map<EntityType<? extends Entity>, Event> ENTITY_LOAD= new HashMap<>()
-        {{
-            EntityType.byString("zombie");
```

Comment:
I encountered class-loading errors unless entity type was called statically like here. This might have changed